src/sim/process.cc

```diff
@@ -401,7 +401,7 @@ Process::unserialize(CheckpointIn &cp)
 }
 
 bool
-Process::map(Addr vaddr, Addr paddr, int size, bool cacheable)
+Process::map(Addr vaddr, Addr paddr, int64_t size, bool cacheable)
```

Comment:
```suggestion
Process::map(Addr vaddr, Addr paddr, Addr size, bool cacheable)
```

I believe it's safe to assume that the size should be unsigned, so we should use an unsigned integer here.

In other part of the code base, we are using Addr for the size of things, we can do it here.

Reply:
Seems reasonable, but if you look at the function that `Process::map` calls, it uses an `int64_t`, not `uint64_t`. Unless we change that function, I think this is safer.

Reply:
Agreed.